src/Components/Institutions/Institutions.jsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import "./Institutions.scss";
-import dp from "../../Assets/Images/nitsLogo.png";
+import dp from "../../../public/images/nitsLogo.png";
```

Comment:
You don't import the images in the public directory. it's been mentioned 100s of times. 

[Read this properly](https://vitejs.dev/guide/assets.html#the-public-directory)